pymystrom/switch.py

```diff
@@ -48,15 +48,21 @@ async def get_state(self) -> None:
         """Get the details from the switch/plug."""
         url = URL(self.uri).join(URL("report"))
         response = await request(self, uri=url)
-        self._consumption = response["power"]
-        self._consumedWs = response["Ws"]
+        try:
+            self._consumption = response["power"]
+        except KeyError:
+            self._consumption = 0
```

Comment:
When does this happen? [`power`](https://api.mystrom.ch/#fbb2c698-e37a-4584-9324-3f8b2f615fe2) should be available for all firmware releases. 

Testet with 2.68.10 (Hardware v1) and 3.82.4 (Hardware v2).

Reply:
WiFi Switch Zero is not covered by myStrom's API docs at the moment and I don't have one to test.

Reply:
I agree, I have the same problem with the new Switch Zero. Have created pull request #34 with my patched version